app/src/main/java/com/cmput301f20t21/bookfriends/repositories/impl/BookRepositoryImpl.java

```diff
@@ -249,10 +249,6 @@ private String getPathStorageFromUrl(String url) {
                 });
     }
 
-    public Task<QuerySnapshot> getDocumentBy(String isbn, String title, String author) {
-        return bookCollection.whereEqualTo("isbn", isbn).whereEqualTo("title", title).whereEqualTo("author", author).get();
-    }
```

Comment:
Not used anywhere(checked with Android Studio) so I deleted it, and it's not practical to do it this way as two books can have the identical isbn, title, and author